gradle-module/src/main/java/org/example/Main.java

```diff
@@ -22,4 +23,11 @@ public static void main(String[] args) {
     private static void test() {
         System.out.println("test");
     }
-}
\ No newline at end of file
+
+   /* asd
+    we
+    asd
+
+    */
+    
+}
```

Comment:
[Category: Consistency] [Severity: Low] There is no newline at the end of the file, which is a common convention in many programming languages, including Java. Adding a newline can prevent potential issues with version control systems and improve compatibility with various tools. Rationale: Following conventions helps maintain consistency across the codebase. [Reference: Java coding conventions]